LiveSplit.AutoSplitters.xml

```diff
@@ -1,5 +1,17 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <AutoSplitters>
+    <AutoSplitter>
+        <Games>
+            <Game>Pepper Grinder</Game>
+        </Games>
+        <URLs>
+            <URL>https://raw.githubusercontent.com/Biksell/asl/main/Pepper%20Grinder/PepperGrinder.asl</URL>
+            <URL>https://github.com/just-ero/asl-help/raw/main/lib/asl-help</URL>
```

Comment:
The linked ASL does not actually use `asl-help` (it's commented out). Are you sure you want to add this?

Reply:
I'm planning on adding signature scanning with `asl-help` once the game updates, so I thought I'd add it so no further PRs are needed for it. Wasn't sure which would be the better practise.